torch_xla/experimental/pjrt.py

```diff
@@ -188,6 +188,9 @@ def process_count() -> int:
 def device_attributes(device: str) -> Dict[str, object]:
   return torch_xla._XLAC._xla_get_device_attributes(device)
 
+@requires_pjrt
+def global_device_attributes() -> List[Dict[str, object]]:
```

Comment:
Could you add a test for this in `test/pjrt/test_experimental_pjrt.py`? Just something simple and backend-agnostic, we need it to pass CI for TPU and CPU